PR: https://github.com/odoo/odoo/pull/44715

From: b0ec3aa1ebb92d4d0100fb3f2a0b97f2a0d364ea
From: qmo-odoo
Date: 2020-02-11 14:36:52

Structural Changes: 3
Total Changes: 6

[REF] event: remove twitter_hashtag and its configuration on event type

This commit removes the field twitter_hashtag from event and event_type.
The justification behind this change is that it has been made obsolete
by our social marketing app and our website builder.

Task ID 2191921
Community PR odoo/odoo#44715
Upgrade PR odoo/upgrade#764

Related: odoo/upgrade#764
Signed-off-by: Thibault Delavallee (tde) <tde@openerp.com>

================================= pseudo patch: =================================

--- a/addons/event/models/event_event.py
+++ b/addons/event/models/event_event.py
@@ -66,8 +66,6 @@ class EventType(models.Model):
     default_timezone = fields.Selection(
         _tz_get, string='Timezone', default=lambda self: self.env.user.tz or 'UTC')
     # communication
-    use_hashtag = fields.Boolean('Use Default Hashtag')
-    default_hashtag = fields.Char('Twitter Hashtag')
     use_mail_schedule = fields.Boolean(
         'Automatically Send Emails', default=True)
     event_type_mail_ids = fields.One2many(
@@ -180,7 +178,6 @@ class EventEvent(models.Model):
         default=lambda self: self.env.company.partner_id,
         domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
     country_id = fields.Many2one('res.country', 'Country',  related='address_id.country_id', store=True, readonly=False)
-    twitter_hashtag = fields.Char('Twitter Hashtag')
     # badge fields
     badge_front = fields.Html(string='Badge Front')
     badge_back = fields.Html(string='Badge Back')
@@ -303,9 +300,6 @@ class EventEvent(models.Model):
             if self.event_type_id.auto_confirm:
                 self.auto_confirm = self.event_type_id.auto_confirm
 
-            if self.event_type_id.use_hashtag:
-                self.twitter_hashtag = self.event_type_id.default_hashtag
-
             if self.event_type_id.use_timezone:
                 self.date_tz = self.event_type_id.default_timezone
 
